test-core/src/test/groovy/org/grails/forge/create/CreateAppSpec.groovy

```diff
@@ -2,16 +2,30 @@ package org.grails.forge.create
 
 import org.grails.forge.application.OperatingSystem
 import org.grails.forge.utils.CommandSpec
+import spock.lang.PendingFeature
```

Comment:
this import is no longer used.

Reply:
Thanks, I apparently missed to clean that up after myself 🧹